### PR TITLE
content: Limits lessons

### PR DESCRIPTION
**Pull request by CoCowBubble**
Limits

### CODE REVIEW

**Review by saumyasinghal747 — APPROVED** [review]
This looks good! You can go ahead and merge :)

File: content/courses/calculus/limits/applications/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+$$1 <\lim_{x\to 0} \frac{\sin{x}}{x} < 1.$$
+This is pretty redundant, and implies that
+$$\lim_{x\to 0}\frac{\sin{x}}{x}=1.$$
+Yes, really - it's just 1. Now, try a two-step limit. Don't forget to use what you just learned! 2.
+$$\lim_{x\to 0}\frac{\tan{x}}{x}$$
+
+<b>Saumya help me I don't know how to insert a user input box, I tried to copy paste the code but it didn't work :( </b>
+
+This is one example of how the limit we just learned can sneak up on you! Here's how we transform it into two problems we can solve:
+$$\lim_{x\to 0}\frac{\tan{x}}{x} =\lim_{x\to 0}\frac{\sin{x}}{x} \cdot \frac{1}{\cos{x}} = \lim_{x\to 0}\frac{\sin{x}}{x} \cdot  \lim_{x\to 0}\frac{1}{\cos{x}} =1\cdot 1=1.$$
```
Comment (saumyasinghal747) [inline content/courses/calculus/limits/applications/index.md]: This line ends up being so long on phones that it runs off the page.
You can break this up by converting
```latex
$$blah blah =  llama's foot = my grandma's clay oven$$
```
to 
```latex
$$blah blah =  llama's foot $$
$$= my grandma's clay oven$$
```

File: content/courses/calculus/limits/applications/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<iframe  src="https://www.desmos.com/calculator/jmllvl6cqk?embed" class="graph" width="500px" height="500px" style="border: 1px solid #ccc" frameborder="0"></iframe>
+
+In the graph above, we have 3 values :
+$$\sin{\theta}, \theta, \tan{\theta}.$$
+If you don't immediately have an answer, think for a few minutes about how we could order these three values in an inequality. One possible (wrong) answer is
+$$\tan{\theta} < \theta < \sin{\theta}.$$
+Record your answer here before continuing:
+</p><input type="text" id="txt_1" onkeyup="saveValue(this);" placeholder="Type your answer here" class="form-control form-control-sm">
+
+<b>Saumya help me I don't know how to insert a user input box, I tried to copy paste the code but it didn't work :( </b>
```
Comment (saumyasinghal747) [inline content/courses/calculus/limits/applications/index.md]: Please remove this!

File: content/courses/calculus/limits/applications/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+$$\lim_{x\to 0}\cos{x}=1.$$
+So, now we say for sure that
+$$1 <\lim_{x\to 0} \frac{\sin{x}}{x} < 1.$$
+This is pretty redundant, and implies that
+$$\lim_{x\to 0}\frac{\sin{x}}{x}=1.$$
+Yes, really - it's just 1. Now, try a two-step limit. Don't forget to use what you just learned! 2.
+$$\lim_{x\to 0}\frac{\tan{x}}{x}$$
+
+<b>Saumya help me I don't know how to insert a user input box, I tried to copy paste the code but it didn't work :( </b>
+
```
Comment (saumyasinghal747) [inline content/courses/calculus/limits/applications/index.md]: And this

**Review by saumyasinghal747 — CHANGES_REQUESTED** [review]
I've added a few comments - just follow up on them and re-request this review when you're done!

File: content/courses/calculus/integral/substitution/index.md [hunk 4]
```diff
@@ -1,7 +1,11 @@
 
 ---
 title: Integration by Substitution
+index: 18
+type: lesson
```
Comment (saumyasinghal747) [inline content/courses/calculus/integral/substitution/index.md:4]: Remember that these are necessary for every single file!

File: content/courses/calculus/limits/applications/index.md (hunk 2 above)
Comment (CoCowBubble) [reply content/courses/calculus/limits/applications/index.md]: I bee-leaf I did this

File: content/courses/calculus/limits/applications/index.md (hunk 3 above)
Comment (CoCowBubble) [reply content/courses/calculus/limits/applications/index.md]: I beef-leave I did this too

File: content/courses/calculus/limits/applications/index.md (hunk 1 above)
Comment (CoCowBubble) [reply content/courses/calculus/limits/applications/index.md]: It has been updated

File: content/courses/calculus/integral/substitution/index.md (hunk 4 above)
Comment (CoCowBubble) [reply content/courses/calculus/integral/substitution/index.md:4]: I don't know if this is a request or if it's just a comment. If it's a request, do I need to approve it?

Comment (saumyasinghal747) [reply content/courses/calculus/integral/substitution/index.md:4]: this  is just a reminder

File: content/courses/math-programming/index.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ type: course
 ---
 
 Often, we come across functions that we do not know how to evaluate without having qualitative knowledge of the input value. For example, $\sin(x)$, $\max(x, y)$[^1] , and $|x|$.
-
+achieve a certain value, but there's no clear formula we can use to evaluate the function.
```
Comment (saumyasinghal747) [inline content/courses/math-programming/index.md]: I think this is a glitch from github, but just remove this line.

**Review by saumyasinghal747 — CHANGES_REQUESTED** [review]
See comment.

File: content/courses/math-programming/index.md (hunk 5 above)
Comment (saumyasinghal747) [reply content/courses/math-programming/index.md]: (line 11, I mean)

**Review by saumyasinghal747 — APPROVED** [review]
Yahoo! Looks good.

**Pull request merged**